MdePkg/Include/Register/Intel/StmApi.h

```diff
@@ -50,12 +53,25 @@ typedef struct {
   ///
   /// The total STM_HEADER should be 4K.
   ///
+  
+  // Pad to take up 1KB of space
+  UINT8     Reserved2[SIZE_1KB - 5 * sizeof (UINT32) - 3 * sizeof (UINT8) - sizeof (UINT16) - sizeof (STM_FEAT)];

```

Comment:
Should this only be `... 2 * sizeof (UINT8) ...`?

Reply:
You're correct.